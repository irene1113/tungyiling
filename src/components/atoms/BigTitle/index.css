.big-title {
	border-top: 1px solid var(--gray);
	border-bottom: 1px solid var(--gray);
	font-family: 'Roboto', sans-serif;
	font-weight: bold;
	font-size: 70px;
	color: var(--black);
}
