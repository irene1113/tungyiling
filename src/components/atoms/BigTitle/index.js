import React from 'react';
import classnames from 'classnames';

import styles from './index.css';

const BigTitle = ({ className, children }) => (
	<div className={classnames(styles.bigTitle, className)}>{children}</div>
);

export default BigTitle;
