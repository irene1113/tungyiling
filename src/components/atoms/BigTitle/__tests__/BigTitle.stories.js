import React from 'react';
import { storiesOf } from '@storybook/react';
import { text } from '@storybook/addon-knobs/react';

import BigTitle from 'components/atoms/BigTitle';

const stories = storiesOf('atoms/BigTitle', module);

stories.add('__interactive', () => <BigTitle>{text('title', 'WEB DEVELOPMENT')}</BigTitle>);
