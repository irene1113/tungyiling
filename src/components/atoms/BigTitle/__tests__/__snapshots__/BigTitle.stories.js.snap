// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="bigTitle"
  >
    WEB DEVELOPMENT
  </div>
</DocumentFragment>
`;
